requestEdges req
---
sensor_msgs/Image edgeImage
